third_party/move/language/tools/move-cli/src/base/new.rs

```diff
@@ -57,6 +57,19 @@ impl New {
             None => Path::new(&name),
         };
         create_dir_all(path.join(SourcePackageLayout::Sources.path()))?;
+        let mut source_file = std::fs::File::create(
+            path.join(SourcePackageLayout::Sources.path())
+                .join(format!("{name}.move")),
+        )?;
+        writeln!(
+            &mut source_file,
+            r#"/*
+/// Module: {name}
+module {name}::{name} {{
```

Comment:
Too many `{`?

Reply:
{} is a placeholder for formatting.
{{ and }} are used to escape curly braces and include them literally.